int16 desire_rpm_right
int16 desire_rpm_left
int16 spray_btn
int16 other_btn
